tiledb/sm/misc/constants.cc

```diff
@@ -137,6 +137,9 @@ const double empty_float64 = std::numeric_limits<double>::quiet_NaN();
 /** The special value for an empty char. */
 const char empty_char = std::numeric_limits<char>::min();
 
+/** The special value for an empty int8. */
+const std::byte empty_byte = std::numeric_limits<std::byte>::min();
```

Comment:
While this may compile, with no comparison function the idea of a minimum is, at best, strained. Better just to initialize to zero. Might as well upgrade to `constexpr` while you're at it.
```suggestion
constexpr std::byte empty_byte{0};
```